Templates/Rulesets/ruleset-empty/.editorconfig

```diff
@@ -197,4 +189,6 @@ dotnet_diagnostic.IDE0069.severity = none
 dotnet_diagnostic.CA2225.severity = none
 
 # Banned APIs
-dotnet_diagnostic.RS0030.severity = error
\ No newline at end of file
+dotnet_diagnostic.RS0030.severity = error
+
+dotnet_diagnostic.OLOC001.license_header = // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.\n// See the LICENCE file in the repository root for full licence text.
```

Comment:
We don't want this in the templates.

Reply:
Oops sorry. Should've checked after copy-pasting.